src/raygun.rum.js

```diff
@@ -155,12 +155,10 @@ var raygunRumFactory = function(window, $, Raygun) {
     };
 
     this.endSession = function() {
-      sendItemImmediately({
-        sessionId: self.sessionId,
-        requestId: self.requestId,
-        timestamp: new Date().toISOString(),
-        type: 'session_end',
-      });
```

Comment:
I think we should still send the `session_end` even if it may not be used. It just makes the intention clear and keeps the API consistent with how RUM for mobile operates.

Reply:
My worry around this is that because it isn't used, it may be forgotten about and changes could be made to the worker that would trigger this event in the data service under the radar

Reply:
After a bit more digging - The worker will only end the session using this event type when there is no session cached, for example, when the worker has restarted.

This would be consistent with other providers, even though the `session_end` event is not utilised properly. I'm happy to do this either way. Thoughts?

Reply:
Spoke in person - going to send this event because it's consistent with the other providers 👍